examples/otlp/src/main/java/io/opentelemetry/example/otlp/OtlpExporterExample.java

```diff
@@ -56,5 +56,7 @@ public static void main(String[] args) throws InterruptedException {
 
     // sleep for a bit to let everything settle
     Thread.sleep(2000);
+
+    System.out.println("Bye");
```

Comment:
Currently with 1.9.1 the program hangs (applies to the jaeger example) instead of shutting down correctly. It shuts down properly with `1.10.0-SNAPSHOT`, which I believe is due to [the change to use daemon threads in the dispatcher](https://github.com/open-telemetry/opentelemetry-java/commit/74fb36b6cb79dd80dd0d57c330f85a110599254c). 

Anyway, having a log line here like the jaeger example was helpful to know that the example had actually done some work.